chainer/utils/conv.py

```diff
@@ -33,6 +33,23 @@ def get_conv_outsize(size, k, s, p, cover_all=False, d=1):
         return (size + p * 2 - dk) // s + 1
 
 
+def get_conv_slices(
+        size, k, s, p, cover_all=False, d=1, include_pad=True, dtype='l'):
```

Comment:
I think it's better to add test case for this.

Reply:
I think it's ok because it's an undocumented function and tested indirectly.
I made it explicitly private.